src/main/java/mate/jdbc/service/impl/DriverServiceImpl.java

```diff
@@ -14,6 +15,14 @@ public class DriverServiceImpl implements DriverService {
 
     @Override
     public Driver create(Driver driver) {
+        List<Driver> drivers = driverDao.getAll();
+        for (Driver driver1 : drivers) {
```

Comment:
`driver1` is a good example of a bad naming)

Reply:
:) The driver is given in incoming parameter, so I believed Idea ))) Fixed